apps/coscrad-frontend/src/components/resources/photographs/photograph-detail.full-view.presenter.tsx

```diff
@@ -0,0 +1,15 @@
+import { IDetailQueryResult, IPhotographViewModel } from '@coscrad/api-interfaces';
+import './photograph-detail.full-view.presenter.css';
+
+export const PhotographDetailFullViewPresenter = ({
+    data: photograph,
```

Comment:
Use destructuring here and eliminate the need for all `photograph.key` notation.

Reply:
Not sure I understand this.  Isn't this already destructured to get the data object out aliased as photograph?

Reply:
Done